Index: vim-engine/src/main/kotlin/com/maddyhome/idea/vim/api/Options.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/vim-engine/src/main/kotlin/com/maddyhome/idea/vim/api/Options.kt b/vim-engine/src/main/kotlin/com/maddyhome/idea/vim/api/Options.kt
--- a/vim-engine/src/main/kotlin/com/maddyhome/idea/vim/api/Options.kt	(revision 2cc7ce5b316be5665406dcf8d3e41116ccbfb0b0)
+++ b/vim-engine/src/main/kotlin/com/maddyhome/idea/vim/api/Options.kt	(date 1706273373741)
@@ -302,7 +302,7 @@
   public val ideaglobalmode: ToggleOption = addOption(ToggleOption("ideaglobalmode", GLOBAL, "ideaglobalmode", false))
   public val ideastrictmode: ToggleOption = addOption(ToggleOption("ideastrictmode", GLOBAL, "ideastrictmode", false))
   public val ideatracetime: ToggleOption = addOption(ToggleOption("ideatracetime", GLOBAL, "ideatracetime", false))
-  public val octopushandler: ToggleOption = addOption(ToggleOption("octopushandler", GLOBAL, "octopushandler", true))
+  public val octopushandler: ToggleOption = addOption(ToggleOption("octopushandler", GLOBAL, "octopushandler", false))
 }

 private class MultikeyMap(vararg entries: Option<VimDataType>) {
